src/dct/systems/IADS.lua

```diff
@@ -175,12 +167,10 @@ function IADS:enableSAM(site)
 			return
 		end
 	end
-
 	site.group:getController():setOption(
 		AI.Option.Ground.id.ALARM_STATE,
 		AI.Option.Ground.val.ALARM_STATE.RED)
 	site.Enabled = true
-	env.info("SAM: "..site.Name.." enabled")
```

Comment:
not sure I understand why we are just removing debug lines like this. Maybe convert to using debug logger messages?

Reply:
They're uneeded and are worse in than out. They don't serve any function for debugging OEO and I can't see why they would elsewhere.

I can readd them later as logger messages, but that's not key to this commit.